tests/cephfs/cephfs_vol_management/cephfs_vol_mgmt_create_vol_component_exist_name.py

```diff
@@ -47,8 +55,45 @@ def run(ceph_cluster, **kw):
         if output1 == 0 or output2 == 0 or output3 == 0:
             return 1
         return 0
-
     except Exception as e:
-        log.info(e)
-        log.info(traceback.format_exc())
+        log.error(e)
+        log.error(traceback.format_exc())
         return 1
+
+
+def create_fs(client, vol_name, build, validate=True, **kwargs):
```

Comment:
can we have this in cephfs_utilsV1.py. as this generic and everyone can use it

Reply:
@AmarnatReddy I can do that , but there is one thing I am concerning. I believe that other functions that have used the function could have conflict with this..? So I am curious that if there is way to find a build number from `client1`? in that way we can implement the 4 and 5 `branch` in the function inside..